claim/README.md

```diff
@@ -261,18 +261,23 @@ If you are certain firewall and proxy settings are not the issue, you should con
 GitHub](https://github.com/netdata/netdata/issues/new?labels=bug%2C+needs+triage%2C+ACLK&template=bug_report.md&title=ACLK-available-is-false)
 with details about your system and relevant output from `error.log`.
 
-### Unclaim (remove) an Agent from Netdata Cloud
+### Unclaim and reclaim a node
 
-The best method to remove an Agent from Netdata Cloud is to unclaim it by deleting the `cloud.d/` directory in your
-Netdata library directory.
+To unclaim a node from your Space in Netdata Cloud, delete the `cloud.d/` directory in your Netdata library directory.
 
 ```bash
 cd /var/lib/netdata   # Replace with your Netdata library directory, if not /var/lib/netdata/
 sudo rm -rf cloud.d/
 ```
 
-Once you delete the `cloud.d/` directory, the ACLK will not connect to Cloud the next time the Agent starts, and Cloud
-will show it as **unreachable**. You can then remove it from your War Rooms.
+This node no longer has access to the credentials it was claimed with and cannot connect to Cloud via the ACLK.
```

Comment:
Should we call it "Netdata Cloud" in contexts like this? I just read your long gif-change PR and the references to "Netdata Cloud" are much more natural and easy to parse.